yt_dlp/extractor/rheinmaintv.py

```diff
@@ -0,0 +1,601 @@
+                'height': 1080,
+                'tbr': 4689,
+                'asr': None,
+                'vcodec': 'H264',
+                'acodec': 'none',
+                'protocol': 'ism',
+                'fragments': 'count:174',
+                'has_drm': False,
+                '_download_params': {
+                    'stream_type': 'video',
+                    'duration': 3475200000,
+                    'timescale': 10000000,
+                    'width': 1920,
+                    'height': 1080,
+                    'fourcc': 'H264',
+                    'language': 'und',
+                    'codec_private_data': '0000000167640028ACD940780227E5C044000003000400000300C83C60C6580000000168EBECB22C',
+                    'sampling_rate': None,
+                    'channels': 2,
+                    'bits_per_sample': 16,
+                    'nal_unit_length_field': 4
+                }
+            }, {
+                'format_id': '5995',
+                'url': 'https://rmtvmedia.streaming.mediaservices.windows.net/ee75c842-1751-44bc-bc16-bd9dcefc4303/CasinoMainz.ism/manifest',
+                'manifest_url': 'https://rmtvmedia.streaming.mediaservices.windows.net/ee75c842-1751-44bc-bc16-bd9dcefc4303/CasinoMainz.ism/manifest',
+                'ext': 'mp4',
+                'width': 1920,
+                'height': 1080,
+                'tbr': 5995,
+                'asr': None,
+                'vcodec': 'H264',
+                'acodec': 'none',
+                'protocol': 'ism',
+                'fragments': 'count:174',
+                'has_drm': False,
+                '_download_params': {
+                    'stream_type': 'video',
+                    'duration': 3475200000,
+                    'timescale': 10000000,
+                    'width': 1920,
+                    'height': 1080,
+                    'fourcc': 'H264',
+                    'language': 'und',
+                    'codec_private_data': '0000000167640028ACD940780227E5C044000003000400000300C83C60C6580000000168EBECB22C',
+                    'sampling_rate': None,
+                    'channels': 2,
+                    'bits_per_sample': 16,
+                    'nal_unit_length_field': 4
+                }
+            }],
+            'subtitles': {},
+            'thumbnail': 'https://rmtvmedia0.blob.core.windows.net/e1cf7655-91b9-4b7e-b19f-a79852c7b41f/06_Standard.jpg?sv=2016-05-31&sr=c&sig=jQPJSdxCxJMX8yUZkFUPNrv1qISb0pCouK9MzAMSOMc%3D&st=2022-11-14T14%3A50%3A02Z&se=3022-11-15T14%3A50%3A02Z&sp=r',
+            'timestamp': 1668527402,
+            'duration': 348.0,
+            'view_count': int,
+            'upload_date': '20221115'
+        }
+    }]
+
+    def _real_extract(self, url):
+        mobj = self._match_valid_url(url)
+        display_id = mobj.group('display_id')
+        video_id = 'vom %s' % mobj.group('date')
+        if mobj.group('serial_number'):
+            video_id += ' (%s)' % mobj.group('serial_number')
```

Comment:
* can be simplified
* Use only `\w-` for id, not spaces/`()`

Reply:
Good point.
If url == os.path.join(contentUrl, video_id) is desired, however, different changes are required.
Done.